Tasks/DeployTestAgent/TestAgentConfiguration.ps1

```diff
@@ -204,6 +204,16 @@ function Set-TestAgentConfiguration
         $yesno = GetBoolAsYesNo($EnableAutoLogon)
         $configArgs = $configArgs + ("/enableAutoLogon:{0}" -f $yesno)
     }
+  
+    if ($PSBoundParameters.ContainsKey('DisableScreenSaver'))
+    {
+        if (-not $configAsProcess)
+        {
+            throw "DisableScreenSaver option is not valid for configureAsService."
```

Comment:
testing : Manual testing to ensure that when run as process, DisableScreenSaver is true and when run as service DisableScreenSaver is not set.